susemanager-utils/susemanager-sls/src/beacons/pkgset.py

```diff
@@ -1,45 +1,49 @@
 # -*- coding: utf-8 -*-
-'''
+"""
 Watch RPM or DPkg database via cookies and fire
 an event to the SUSE Manager if that has been changed.
 
 Author: Bo Maryniuk <bo@suse.de>
-'''
+"""
 
 from __future__ import absolute_import
-from salt.utils.odict import OrderedDict
 import os
-import logging
-log = logging.getLogger(__name__)
 
+import salt.cache
+import salt.config
 
-__virtualname__ = 'pkgset'
 
-PKG_PLUGINS = OrderedDict([
-        ("/usr/lib/zypp/plugins/commit/venv-zyppnotify", "/var/cache/venv-salt-minion/rpmdb.cookie"),
-        ("/usr/share/yum-plugins/venv-yumnotify.py",     "/var/cache/venv-salt-minion/rpmdb.cookie"),
-        ("/usr/bin/venv-dpkgnotify",                     "/var/cache/venv-salt-minion/dpkg.cookie"),
-        ("/usr/lib/zypp/plugins/commit/zyppnotify",      "/var/cache/salt/minion/rpmdb.cookie"),
-        ("/usr/share/yum-plugins/yumnotify.py",          "/var/cache/salt/minion/rpmdb.cookie"),
-        ("/usr/bin/dpkgnotify",                          "/var/cache/salt/minion/dpkg.cookie")
-    ])
-COOKIE_PATH = None
+__virtualname__ = "pkgset"
+
+__opts__ = salt.config.minion_config('/etc/salt/minion')
```

Comment:
We need to take care of the `venv-salt-minion` case here right?

Reply:
Oh, yes, correct. Will adjust it for it.